[DOC] Updated mirror maker docs (new options added) (#1768)
strimzi/strimzi-kafka-operator: documentation/book/appendix_crds.adoc

[options="header"]
|====
|Property                     |Description
|numStreams            1.2+<.<|Specifies the number of consumer stream threads to create.
|integer
|offsetCommitInterval  1.2+<.<|Offset commit interval in ms. Default value is 60 000.
|integer
|groupId               1.2+<.<|A unique string that identifies the consumer group this consumer belongs to.
|string

> |bootstrapServers      1.2+<.<|A list of host:port pairs to use for establishing the initial connection to the Kafka cluster.

Reviewer: ```suggestion
|bootstrapServers      1.2+<.<|A list of host:port pairs for establishing the initial connection to the Kafka cluster.
```

|string
|authentication        1.2+<.<|Authentication configuration for connecting to the cluster. The type depends on the value of the `authentication.type` property within the given object, which must be one of [tls, scram-sha-512, plain].
|xref:type-KafkaMirrorMakerAuthenticationTls-{context}[`KafkaMirrorMakerAuthenticationTls`], xref:type-KafkaMirrorMakerAuthenticationScramSha512-{context}[`KafkaMirrorMakerAuthenticationScramSha512`], xref:type-KafkaMirrorMakerAuthenticationPlain-{context}[`KafkaMirrorMakerAuthenticationPlain`]

> |config                1.2+<.<|The mirror maker consumer config. Properties with the following prefixes cannot be set: ssl., bootstrap.servers, group.id, sasl., security.

Reviewer: ```suggestion
|config                1.2+<.<|The Mirror Maker consumer config. Properties with the following prefixes cannot be set: ssl., bootstrap.servers, group.id, sasl., security.
```

|map

> |tls                   1.2+<.<|TLS configuration for connecting to the cluster.

Reviewer: ```suggestion
|tls                   1.2+<.<|TLS configuration for connecting Mirror Maker to the cluster.
```

[options="header"]
|====
|Property                   |Description

> |bootstrapServers    1.2+<.<|A list of host:port pairs to use for establishing the initial connection to the Kafka cluster.

Reviewer: ```suggestion
|bootstrapServers    1.2+<.<|A list of host:port pairs for establishing the initial connection to the Kafka cluster.
```

|string

> |abortOnSendFailure  1.2+<.<|Configure the mirror maker to exit on a failed send. Default value is `true`.

Reviewer: ```suggestion
|abortOnSendFailure  1.2+<.<|Flag to set the Mirror Maker to exit on a failed send. Default value is `true`.
```

|boolean
|authentication      1.2+<.<|Authentication configuration for connecting to the cluster. The type depends on the value of the `authentication.type` property within the given object, which must be one of [tls, scram-sha-512, plain].
|xref:type-KafkaMirrorMakerAuthenticationTls-{context}[`KafkaMirrorMakerAuthenticationTls`], xref:type-KafkaMirrorMakerAuthenticationScramSha512-{context}[`KafkaMirrorMakerAuthenticationScramSha512`], xref:type-KafkaMirrorMakerAuthenticationPlain-{context}[`KafkaMirrorMakerAuthenticationPlain`]

> |config              1.2+<.<|The mirror maker producer config. Properties with the following prefixes cannot be set: ssl., bootstrap.servers, sasl., security.

Reviewer: ```suggestion
|config              1.2+<.<|The Mirror Maker producer config. Properties with the following prefixes cannot be set: ssl., bootstrap.servers, sasl., security.
```

|map

> |tls                 1.2+<.<|TLS configuration for connecting to the cluster.

Reviewer: ```suggestion
|tls                 1.2+<.<|TLS configuration for connecting Mirror Maker to the cluster.
```
